packages/flutter/lib/src/material/date_picker.dart

```diff
@@ -1134,18 +1143,23 @@ Future<DateTimeRange?> showDateRangePicker({
   );
 }
 
+/// Returns a string describing a date
+///
+/// If `dateFormat` is null, then it uses `localizations` to format the date
```

Comment:
Comments must always end with a period.

Reply:
Thank you for letting me know! I have fixed this now.